config/deployment/apiservice.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: apiregistration.k8s.io/v1
+kind: APIService
+metadata:
+  name: v1.organization.appuio.io
+spec:
+  insecureSkipTLSVerify: true
```

Comment:
Out of curiosity: what's the approach to take to avoid requiring `insecureSkipTLSVerify=true` for an APIService?

Reply:
I will provide the option in the component.

We can provide custom key/certificate and them provide a `caBundle` as part of this `APIServer` resource. I did not try to implement this for this sample deployment, as I did not think this was useful for the local environment.